package.json

```diff
@@ -11,11 +11,16 @@
     "prepare": "is-ci || husky install",
     "start": "next start",
     "storybook": "storybook dev -p 6006",
-    "typecheck": "tsc --noEmit"
+    "typecheck": "tsc --noEmit",
+    "supabase:login": "supabase login",
+    "supabase:types": "supabase gen types typescript --project-id zcetythxtznisfgaakyk > types/database.ts"
   },
   "prettier": "@vercel/style-guide/prettier",
   "dependencies": {
     "@radix-ui/themes": "^2.0.2",
+    "@supabase/auth-helpers-nextjs": "^0.8.7",
```

Comment:
@okanisildar I guess this is deprecated in favor of `ssr` https://supabase.com/docs/guides/auth/server-side/migrating-to-ssr-from-auth-helpers